# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/utils/PerformanceRecorder.cpp
# Commit: e85b278387fc
# Full Hash: e85b278387fc5259efead947f5069c9c2dcbffad
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-11 21:56:57
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Pass TrackingId instead of nsCString to CaptureStage. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165824
# ==============================================================================

diff -r 0fa64a630cc7 -r e85b278387fc dom/media/utils/PerformanceRecorder.cpp
--- a/dom/media/utils/PerformanceRecorder.cpp	Wed Jan 11 11:47:45 2023 +0000
+++ b/dom/media/utils/PerformanceRecorder.cpp	Wed Jan 11 11:47:45 2023 +0000
@@ -269,7 +269,7 @@
     };
     mName = Some(nsPrintfCString(
         "CaptureVideoFrame %s %dx%d %s %s", mSource.Data(), mWidth, mHeight,
-        imageTypeToStr(mImageType), mTrackingId.get()));
+        imageTypeToStr(mImageType), mTrackingId.ToString().get()));
   }
   return *mName;
 }